src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2514,5 +2514,53 @@ public SafeHandle NativeHandle
                 return new NUI.SafeNativeWindowHandle(this);
             }
         }
+
+        /// <summary>
+        /// It is used to make a given window a modal one. 
+        /// Once it's set as a modal window, only that window can receive user input.
+        /// </summary>
+        /// <param name="modal"> If modal is true, set the modal window or unset.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetModal(bool modal)
+        {
+            Interop.Window.SetModal(SwigCPtr, modal);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Gets whether the window is modal or not.
+        /// </summary>
+        /// <returns>Returns true if the full screen sized window is.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsModal()
+        {
+            bool ret = Interop.Window.IsModal(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// This function is designed to keep a specific window always on top of other windows. 
+        /// Its behavior is effective among windows with the same notification level.
+        /// </summary>
+        /// <param name="alwaysTop"> If alwaysTop is true, set the topmost window and keeping or unset.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetAlwaysTop(bool alwaysTop)
+        {
+            Interop.Window.SetAlwaysTop(SwigCPtr, alwaysTop);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Gets whether the window is always top or not.
+        /// </summary>
+        /// <returns>Returns true if the window is always top is.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsAlwaysTop()
+        {
+            bool ret = Interop.Window.IsAlwaysTop(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
```

Comment:
ditto

```suggestion
        /// <summary>
        /// Gets or sets a value indicating whether the window is alwats on top of other windows or not.
        /// </summary>
        [EditorBrowsable(EditorBrowsableState.Never)]
        public bool IsAlwaysTop
        {
            get
            {
                bool ret = Interop.Window.IsAlwaysTop(SwigCPtr);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                return ret;
            }
            set
            {
                Interop.Window.SetAlwaysTop(SwigCPtr, alwaysTop);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
            }
        }
```

Reply:
How about rename to `public bool Topmost { get; set; }`
It is API of uwp

https://learn.microsoft.com/en-us/dotnet/api/system.windows.window.topmost?view=windowsdesktop-8.0&viewFallbackFrom=dotnet-uwp-10.0

Reply:
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.windowing.overlappedpresenter.isalwaysontop?view=windows-app-sdk-1.5#microsoft-ui-windowing-overlappedpresenter-isalwaysontop

Reply:
Actually I considered the "Topmost". But this word is already used in KeyGrabMode.
I'd better than AlwaysOnTop. 
Thank you for kindly review. :D